File: python/api-calls/media_analyze_post.py
#!/usr/bin/env python

import os
import requests

url = "https://api.dolby.com/media/analyze"
headers = {
    "x-api-key": os.environ["DOLBYIO_API_KEY"],
    "Content-Type": "application/json",
    "Accept": "application/json",
}

body = {
    "input": os.environ["DOLBYIO_INPUT"],
}

response = requests.post(url, json=body, headers=headers)
response.raise_for_status()
print(response.json()["job_id"])

File: javascript/api-calls/media_analyze_post.js
#!/usr/bin/env node

const axios = require('axios').default;

const config = {
    method: 'post',
    url: 'https://api.dolby.com/media/analyze',
    headers: {
        'x-api-key': process.env.DOLBYIO_API_KEY,
        'Content-Type': 'application/json',
        'Accept': 'application/json'
    },
    data: {
        input: 'dlb://in/example.mp3'
    }
};

axios(config)
    .then(function (response) {
        console.log(response.data.job_id);
    })
    .catch(function (error) {
        console.log(error);
    });

